runk/cmd/sbox/main.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
if sbox is a subpackage of runk it doesn't make sense to have runk package at all, or the project should be called runk.